packages/arb-avm-cpp/avm_values/src/buffer.cpp

```diff
@@ -159,7 +168,7 @@ std::vector<RawBuffer> RawBuffer::serialize(
     if (level > 0) {
         value_vector.push_back(1);
         for (uint64_t i = 0; i < NODE_SIZE; i++) {
-            uint256_t hash_ = hash2(123, (*node)[i].hash());
+            uint256_t hash_ = hash2(1234, (*node)[i].hash());
```

Comment:
Presumably the maxAccess field should also affect the hash.  Use that instead of `1234`?

Reply:
This is for serializing inner nodes in the tree, so there's no `maxAccess` for them. I'll try to make that clearer.